### PR TITLE
Revised definition based on working group collaboration

### PR DESCRIPTION
**Pull request by CoralineAda**

### CODE REVIEW

File: content/definition.md [hunk 1]
```diff
@@ -5,10 +5,11 @@ template = "definition"
...
-1. Developed in public view and welcoming of community contributions.
-1. The community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. Creators have the right to prohibit use by individuals or organizations engaged in human rights violations or other behavior deemed unethical.
-1. Creators have the right to solicit reasonable and voluntary compensation from the communities or institutions that benefit from the software.
+1. It is distributed with source code and can be freely used or combined with other software.
+1. It is developed or maintained in public view and welcoming of public  contributions.
+1. Its community of contributors is governed by a code of conduct that is consistently and fairly enforced.
+1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that would violate any person’s human rights including, but not limited to, those enumerated in the Universal Declaration of Human Rights.
```
Comment (nateberkopec) [inline content/definition.md]: What about dual-licensing?

**Comment by CoralineAda** [comment]
Can you say more?
> What about dual-licensing?
>

**Comment by nateberkopec** [comment]
Just pointing out that "It is distributed under a license that..." seems to open the door to dual-licensing a project under an Ethical Source license and a non-Ethical license. Maybe "It is only distributed under a license..." would make intent more clear? Unless dual-licensing with a non-ethical license is OK by the working group.

File: content/definition.md [hunk 2]
```diff
@@ -5,10 +5,11 @@ template = "definition"
 
 Software is considered Ethical Source if it meets the following criteria:
 
-1. Freely distributed with source code and can be used or combined with other software without a royalty or fee.
-1. Developed in public view and welcoming of community contributions.
-1. The community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. Creators have the right to prohibit use by individuals or organizations engaged in human rights violations or other behavior deemed unethical.
-1. Creators have the right to solicit reasonable and voluntary compensation from the communities or institutions that benefit from the software.
+1. It is distributed with source code and can be freely used or combined with other software.
+1. It is developed or maintained in public view and welcoming of public  contributions.
```
Comment (UncannyBingo) [inline content/definition.md]: Two spaces between "public" and "contributions!

Comment (UncannyBingo) [reply content/definition.md]: Do we need to be clearer on what "welcoming" means? Or are we happy for the moment to leave it somewhat vague, given the CoC requirement?

File: content/definition.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Software is considered Ethical Source if it meets the following criteria:
 1. It is distributed with source code and can be freely used or combined with other software.
 1. It is developed or maintained in public view and welcoming of public contributions.
 1. Its community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that would violate any person’s human rights including, but not limited to, those enumerated in the Universal Declaration of Human Rights.
+1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good.
```
Comment (nateberkopec) [inline content/definition.md:11]: Oh interesting - so the idea is that the author specifies the framework? That makes more sense than trying to pick something "universal".

Comment (CoralineAda) [reply content/definition.md:11]: Yeah, that's the idea! This opens the way for licenses like Atmosphere, 966.ICU, and similar to fit under the umbrella.

File: content/definition.md (hunk 2 above)
Comment (CoralineAda) [reply content/definition.md]: Tha latter IMO

**Pull request merged**